language_service/src/hover.rs

```diff
@@ -51,6 +51,27 @@ struct HoverGenerator<'a> {
 }
 
 impl<'a> Handler<'a> for HoverGenerator<'a> {
+    fn at_attr_ref(&mut self, name: &'a ast::Ident) {
+        let description = match name.name.as_ref() {
+                "Config" => "Provides pre-processing information about when an item should be included in compilation.
```

Comment:
There's Attr enum in qsharp\compiler\qsc_hir\src\hir.rs with these choices and almost the same text in comments. I would expect this information to be next to that enum. Does language_service share anything with the compiler? If so, consider moving it close to that enum.

Reply:
Good thinking. That way if something needs to change with the attributes, all the information is in one place (the hir).